pipeline/leech/main.go

```diff
@@ -112,6 +113,8 @@ func realMain(ctx context.Context) error {
 	}
 
 	var event leech.EventRecord
+	// I know its not really the threat model, but it does seem to be a bad habit to use
+	// string replacement for query building.
```

Comment:
The go bigquery connector for Apache Beam doesn't support bind parameters so there isn't much choice unless we write our own BQ integration.